node/src/chain/acceptor.rs

```diff
@@ -228,6 +229,33 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
                 Ok(txs)
             })?;
 
+            for iter in 0..blk.header().iteration {
+                let mut prov = provisioners_list.clone();
+                let committee_keys =
+                    prov.create_committee(&sortition::Config {
+                        committee_size: SELECTION_COMMITTEE_SIZE,
+                        round: blk.header().height,
+                        seed: blk.header().seed,
+                        step: iter * 3,
+                    });
+                if committee_keys.len() != 1 {
+                    error!(
+                        "Unable to generate voting committee for missed block"
+                    )
+                } else {
+                    let generator = committee_keys
+                        .first()
+                        .expect("committee to have 1 entry")
+                        .to_full_bs58();
+                    warn!(
+                        event = "missed iteration",
+                        height = blk.header().height,
+                        iter,
+                        generator,
+                    );
+                }
+            }
+
```

Comment:
nitpick: Could you move this block in a separate function for better readability? I try to keep `try_accept_block` lean.